### PR TITLE
Add Farsi language to the About page

### PR DESCRIPTION
**Pull request by mehr32**
## What does this PR do?
Adds Persian language to the About page

## Why is this change important?
To expand as much as possible

## How to test this PR locally?
Copy the files from this commit to the languages ​​path of the About page

### CODE REVIEW

**Comment by return42** [comment]
Hi @mehr32 thaks a lot for the translation .. to get it work, I had to rename the folder from `fa-IR` to `fa_IR` / I forced pushed my modification on you branch --> [see diff](https://github.com/searxng/searxng/compare/cf0b151731741913f6dba0268caf7032b4114867..34f7abbc16fd8fffea6fadd9c0c01cb48ea1e520)

Now you can test your translation by a `make run` .. I set my browser's locale to fa-IR to get the translated about page ..

In the screenshot below you can see that some hyperlinks are not rendered .. IDK why .. may be its because the Markdown does not like Farsi characters in the links or its related to the r2l notation.

Can you please have a look, may you have a clue why not all links are rendered well / thanks :+1:

image

**Comment by dalf** [comment]
That's because in `[موتور فراجستجو]`, `[` is actually `]`. You can see that if you copy paste the bracket from https://github.com/searxng/searxng/pull/2695/files (it won't show this, if you copy paste the brackets from this comment).

In the `[#searxng:matrix.org]`, `[` is `[`. 

So I guess the fix is to copy paste the brackets another place, and this is most probably a bug in Markdown rendering.

**Comment by return42** [comment]
> That's because in `[موتور فراجستجو]`, `[` is actually `]`.
> In the `[#searxng:matrix.org]`, `[` is `[`. 

~Does it mean the last character before a bracket determines the *direction* of a open or close bracket?  In my Emacs (which supports RTL), when I open a bracket typing key `[` a `]` is inserted / and vice versa .. but I tried different markups but did not get a link working when there are RTL characters in .. can you give an example of a markup that works .. thanks!~

**Comment by mehr32** [comment]
Well, it's great, unfortunately, I couldn't test it and I didn't see any problems. I'll try it and give you the modified version.

**Comment by mehr32** [comment]
I updated and now it works for me. I also translated search-syntax.md. After applying this translation, I will also request money for it

File: searx/infopage/fa_IR/about.md [hunk 1]
```diff
@@ -44,31 +44,31 @@ SearXNG از [OpenSearch] پشتیبانی می کند. برای اطلاعات
...
-[اسناد SearXNG] : {{get_setting('brand.docs_url')}}
+[اسناد SearXNG]: {{get_setting('brand.docs_url')}}
 [searx]: https://github.com/searx/searx
 [موتور فراجستجو]: https://fa.wikipedia.org/wiki/Metasearch_engine
 [وب سایت]: https://translate.codeberg.org/projects/searxng/
 [پروژه Seeks]: https://beniz.github.io/seeks/
 [OpenSearch]: https://github.com/dewitt/opensearch/blob/master/opensearch-1-1-draft-6.md
-[Firefox]: https://support.mozilla.org/en-US/kb/add-or-remove-search-engine-firefox
+[فایرفاکس]: https://support.mozilla.org/en-US/kb/add-or-remove-search-engine-firefox
 [Microsoft Edge]: https://support.microsoft.com/en-us/help/4028574/microsoft-edge-change-the-default-search-engine
```
Comment (return42) [inline searx/infopage/fa_IR/about.md]: "Firefox" is a product name comparable to "Microsoft Edge" and "Chrome" .. I think we should use the product name and not a translation.

**Review by return42 — CHANGES_REQUESTED** [review]
> I updated and now it works for me.

Yeah, `about.md` looks quite good :)

I left a remark in the code .. can you fix it / thanks a lot. 

> I also translated search-syntax.md. 

File `search-syntax.md` is not in this commit .. do you send a new PR for the translation of `search-syntax.md`?

> After applying this translation, I will also request money for it

just to verify:  you did kidding me .. right?

File: searx/infopage/fa_IR/about.md (hunk 1 above)
Comment (mehr32) [reply searx/infopage/fa_IR/about.md]: It is not a translation, in reality it is just a translation with Persian calligraphy. The translation would be something like روباه آتشین But to coordinate with the rest of the browsers, it is fixed

**Comment by mehr32** [comment]
---- File search-syntax.md is not in this commit .. do you send a new PR for the translation of search-syntax.md?
I am a newbie, I honestly don't know how to request money for several commits at once :) If possible, add this file so that I can insert the translation here.
---- just to verify: you did kidding me .. right?
She! A funny mistake! In Persian, پول means money. I meant to make it "push" and because it is pronounced close to پول in Farsi, that's why the translator translated it wrongly!

How should I quote?

**Review by return42 — APPROVED** [review]

**Comment by return42** [comment]
> It is not a translation, in reality it is just a translation with Persian calligraphy .. But to coordinate with the rest of the browsers, it is fixed

Thanks a lot! .. the About page is perfect --> I'm going to merge this PR.

> I am a newbie, I honestly don't know how to

No problem .. just open a new PR.

**Pull request merged**